### DISCUSSION TITLE
How do the date based counts work and/or why don't Immich points show up?

### DISCUSSION THREAD

**Discussion by chrisl8** [Freika/dawarich#314] [Q&A]
I have this in my stats page:
![image](https://github.com/user-attachments/assets/df45becf-089b-4085-8ed4-8b5a1e8702dd)

But if I go to the Map page 2023 has nothing:
![image](https://github.com/user-attachments/assets/f6983f41-3b6e-4d4b-ae58-77e9cead3f70)

And just to be clear, clicking on 2023 shows NO points on the map too, while older years with good Google imports do.

I think that the 2023 data all came from Immich if that helps, although I'm not 100% sure.

Any ideas on what is going on here?

**Comment by Freika** [comment]
I think this is a bug already reported, I'll have a look at it in my spare time

**Comment by fengyuclub** [comment]
I also encountered the same problem. When I first deployed the test, I could filter the corresponding points from the date picker. However, for other reasons, I was preparing to import a large amount of my GPS data. I redeployed dawarich, but the imported points could not be arranged by date. I don’t know if it was because there were too many points.
